### ISSUE TITLE
A use case that seems to allude the seq2seq learner's capability

### ISSUE THREAD

**Issue by dtp555-1212** [keras-team/keras-io#374]
Hello, I recently downloaded and tried your lstm_seq2seq translator. The part that interested me most was the character level processing.

I replaced the training material with my own data. I was trying to see if the learner could learn the concept of test→xml or xml→text. e.g. test	→ &lt;x>test&lt;/x&gt; OR &lt;x&gt;test&lt;/x&gt; → test
Neither of the concepts were captured via the learning process. (BTW: if there was no space, in either the source or target, that crashes the code… ).

Do you think it is ‘possible’ for a neural network to learn such a distinction? (e.g. either to remove the tags when going from XML to text OR more importantly associating some tags when going from text to XML)... If it is possible, can you point me in the right direction to a way to make this work.

Thanks

**Comment by dtp555-1212** [comment]
an update.... on a small dataset, I was able to get it to function only by duplicating all the data. XML->text accuracy resolved pretty quick, but text->XML the 'accuracy' number plateaued (but it seemed to work ok with the small dataset, given a large number of epochs. Is there a way to not have to duplicate that data? (The ultimate dataset is likely to be large, so artificially having to duplicate all the data would not be optimal.) P.S. Since the training data is perfectly balanced and has no ambiguities, I would have hoped (somewhat expected) the accuracy would indicated it had perfected. If you have a suggestion on that, that would be great too.

Thanks

**Comment by sachinprasadhs** [comment]
Hi, 

The tutorial you are referring to has been updated since you opened this issue. 
Could you please refer to this commit https://github.com/keras-team/keras-io/commit/7d9a8190d134b15aa8fb54b457e8ba6c1cb562a3 and run the latest updated Tutorial and let us know if that addresses your concern. Thanks!

If you are still facing an error, please provide the sample reproducible code.

**Comment by sachinprasadhs** [comment]
Hi, 

The tutorial has been updated to latest Keras 3 and it is showing good accuracy, could you please verify and close the issue.

https://keras.io/examples/nlp/lstm_seq2seq/
